controllers/license/deploy/manifests/secret.yaml.tmpl

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: license-secret
+  namespace: license-system
+stringData:
+  MONGO_URI: "{{ .MONGO_URI }}"
```

Comment:
If the namespace cannot be unified, maybe we should use the same secret to store common variables.